packages/shared/styles/components/SfBullets.scss

```diff
@@ -0,0 +1,29 @@
+@import '../variables';
+
+$bullet--border-radius: 50%;
```

Comment:
1. ok, I see, my mistake with double hyphen.
2. The problem with <code>$bullet**s**-</code> vs <code>$bullet-</code> in this case is that the component indeed consist of multiple bullets, but the styling is for single bullet, thus I decided to use singular <code>$bullet-...</code>.
So either I can leave it as is, or change the wrapper class and then use for singular bullet styling the common convention as you advised.

Reply:
Got it, actually, following BEM I think it would be `bullets__item`, right? but then the class name may be bigger then necessary (which is often the case with BEM)...
I like to follow conventions to be predictable as possible, but I understand your position, I think it's more or less the same case as the `hero-item`, we can agree in this way.